app/views/shared/_message.html.haml

```diff
@@ -0,0 +1,14 @@
+-# TODO: 分岐が汚すぎるからどうにかする
+- if (defined?(current_mentor) && current_mentor&.id == message.user_id) || (defined?(current_student) && current_student&.id == message.user_id) || (!defined?(current_mentor) && !defined?(current_student))
```

Comment:
__[HAML-Lint]__

Line is too long. [206/80]


<!-- Comment by [SideCI](https://sideci.com) -->